packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -43,9 +43,12 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
     } catch (e) {
       console.error('Failed to get current card account', errorHandler(e))
       const eligibleAccounts = yield select(selectors.components.debitCard.getEligibleAccounts)
-      yield put(
-        A.getCurrentCardAccountFailure(!isEmpty(eligibleAccounts) ? eligibleAccounts[0] : null)
-      )
+      if (!isEmpty(eligibleAccounts)) {
+        // In case of failure it is set the default account as current
+        yield put(A.getCurrentCardAccountSuccess(eligibleAccounts[0]))
+      } else {
+        yield put(A.getCurrentCardAccountFailure('Could not get user funds'))
```

Comment:
Should the `Could not get user funds ` text be translated too?

Reply:
Is not being shown to the user